precompiles/staking/staking_test.go

```diff
@@ -394,6 +401,65 @@ func allowStaking(t *testing.T, ts testSuite, amount *big.Int) {
 	require.True(t, allowed)
 }
 
+func stakeThroughCosmosAPI(
+	t *testing.T,
+	ctx sdk.Context,
+	bankKeeper bankkeeper.Keeper,
+	stakingKeeper stakingkeeper.Keeper,
+	validator stakingtypes.Validator,
+	staker sdk.AccAddress,
+	amount math.Int,
+) {
+	// Coins to stake with default cosmos denom.
+	coins := sdk.NewCoins(sdk.NewCoin("stake", amount))
+
+	err := bankKeeper.MintCoins(ctx, fungibletypes.ModuleName, coins)
+	require.NoError(t, err)
+
+	err = bankKeeper.SendCoinsFromModuleToAccount(ctx, fungibletypes.ModuleName, staker, coins)
+	require.NoError(t, err)
+
+	shares, err := stakingKeeper.Delegate(
+		ctx,
+		staker,
+		coins.AmountOf(coins.Denoms()[0]),
```

Comment:
Why not just use the string , can define `stake` as a constant ?

Reply:
Sorry, this method is out of scope as it was introduced in another PR.